test/multiapi/Expected/AcceptanceTests/Multiapi/multiapi/aio/_configuration_async.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         super(MultiapiServiceClientConfiguration, self).__init__(**kwargs)
 
         self.credential = credential
-        self.credential_scopes = []
+        self.credential_scopes = ['https://management.azure.com/.default']
```

Comment:
The reason for these mgmt changes in the old multiapi generation, is because I incorrectly had the flag as `azure`, and have fixed it to be `azure-arm`